projects/plugins/jetpack/extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -56,14 +65,16 @@ function load_assets( $attr, $content ) {
 		? get_current_blog_id()
 		: Jetpack_Options::get_option( 'id' );
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
-	$classes         = Blocks::classes( FEATURE_NAME, $attr );
-	$amp_form_action = sprintf( 'https://public-api.wordpress.com/rest/v1.1/sites/%s/email_follow/amp/subscribe/', $blog_id );
-	$is_amp_request  = Blocks::is_amp_request();
+	$wrapper_attributes = \WP_Block_Supports::get_instance()->apply_block_supports();
+	$classes            = ! empty( $wrapper_attributes['class'] ) ? $wrapper_attributes['class'] : '';
+	$styles             = ! empty( $wrapper_attributes['style'] ) ? $wrapper_attributes['style'] : '';
+	$amp_form_action    = sprintf( 'https://public-api.wordpress.com/rest/v1.1/sites/%s/email_follow/amp/subscribe/', $blog_id );
+	$is_amp_request     = Blocks::is_amp_request();
 
 	ob_start();
 	?>
 
-	<div class="<?php echo esc_attr( $classes ); ?>" data-blog-id="<?php echo esc_attr( $blog_id ); ?>">
+	<div class="<?php echo esc_attr( $classes ); ?>" style="<?php echo esc_attr( $styles ); ?>" data-blog-id="<?php echo esc_attr( $blog_id ); ?>">
```

Comment:
How about preventing the entire style attribute from rendering when there are no styles set?